javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js

```diff
@@ -8,8 +8,8 @@ const BasicCourseTable = ( {classes} ) => {
   const rowStyle = (row, _rowIndex) => {
     return  (row.section % 100 === 0)? {backgroundColor: '#CEDEFA'}: {backgroundColor: '#EDF3FE'};
   }
-  const dataAlignment = (_cell, row) => {
-    const alignmnet = (row.section % 100 === 0)? 'left': 'right';
+  const dataAlignment = () => {
+    const alignmnet = 'left';
     return alignmnet
```

Comment:
This isn't an issue that was created by your team, but can we fix the spelling of the variable name "alignment" here?